README.rst

```diff
@@ -118,14 +120,19 @@ The following querystring parameters are supported.
   A list of distros in ``distro.distro_version`` or ``distro`` format.
   i.e. ``?distros=ubuntu, centos.7``
 
-- ``common_sha1``
-  Requires that the repos return be built for a common sha1
-  across all given distros. If a common built sha1 is not found
-  for distros then no results are returned.
-  i.e. ``?common_sha1=True&distros=centos.7,ubuntu.xenial``
+- ``sha1``
```

Comment:
I don't think so, because I think it's pretty common functionality for a search to return all values for a filter if that filter is not applied.